### PR TITLE
kops: Fix {{E2E_NAME}}

### PR DESCRIPTION
**Pull request by zmerlynn**
Injected in #1209

Along the way: Remove env variables we don't need

### CODE REVIEW

**Comment by k8s-oncall** [comment]
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1218)

**Comment by zmerlynn** [comment]
Merging to get jobs back up.

File: jobs/ci-kubernetes-e2e-kops-aws-updown.sh [hunk 1]
```diff
@@ -51,26 +51,14 @@ export CLOUDSDK_COMPONENT_MANAGER_DISABLE_UPDATE_CHECK=true
 # Use default component update behavior
 export CLOUDSDK_EXPERIMENTAL_FAST_COMPONENT_UPDATE=false
 
-# AWS variables
-export KUBE_AWS_INSTANCE_PREFIX="${E2E_NAME}"
```
Comment (ixdy) [inline jobs/ci-kubernetes-e2e-kops-aws-updown.sh]: I'm guessing this isn't used by kops?

Comment (zmerlynn) [inline jobs/ci-kubernetes-e2e-kops-aws-updown.sh]: Yeah, the kops config uses an explicit DNS hosted zone (in E2E_NAME) that it then populates.

**Pull request merged**